.env.production

```diff
@@ -1,6 +1,6 @@
-EXPENSIFY_URL_CASH=https://new.expensify.com/
+NEW_EXPENSIFY_URL=https://new.expensify.com/
 EXPENSIFY_URL_SECURE=https://secure.expensify.com/
```

Comment:
Let's change it to `SECURE_EXPENSIFY_URL` everywhere?

Reply:
UPDATED